### PR TITLE
Volunteer profile fix

### PR DESCRIPTION
**Pull request by BrianRamsay**
A branch for resolving the merge conflicts with development

### CODE REVIEW

File: tests/code/test_participants.py [hunk 1]
```diff
@@ -213,9 +213,10 @@ def test_sendUserData():
     # Tests the Kiosk
     # user is banned
     signedInUser, userStatus = sendUserData("B00739736", 2, 1)
-    assert userStatus == "banned"
-    with pytest.raises(DoesNotExist):
-        EventParticipant.get(EventParticipant.user==signedInUser, EventParticipant.event==2)
+    assert signedInUser, userStatus == "banned"
```
Comment (BrianRamsay) [inline tests/code/test_participants.py]: This is confusing. How does `assert` handle this? Are you testing that `signedInUser` is true, that `userStatus == "banned"`, or both?

Even if it works it's not clear what the intention is. If you want to test both, use two assert statements.

**Review by BrianRamsay — COMMENTED** [review]
So close! Just one comment on a test.

File: tests/code/test_participants.py (hunk 1 above)
Comment (gastonjarju) [reply tests/code/test_participants.py]: Added a transaction to the test function and also realized that the test database was missing some endDates for the banned user. Thus, I added a ban endDate to it as well.

Comment (BrianRamsay) [reply tests/code/test_participants.py]: This isn't actually resolved. They are still both on the same line. Do you need to commit and push some changes?

**Review by BrianRamsay — COMMENTED** [review]
I think you might be missing a recent change.

File: tests/code/test_participants.py (hunk 1 above)
Comment (gastonjarju) [reply tests/code/test_participants.py]: Commit and pushed!

**Comment by BrianRamsay** [comment]
Woohoo! Merging.

**Pull request merged**